internal/ogc/features/datasources/geopackage/metadata.go

```diff
@@ -191,6 +192,12 @@ func readSchema(db *sqlx.DB, table featureTable, fidColumn, externalFidColumn st
 		collectionNames = append(collectionNames, collection.ID)
 	}
 
+	// if table gpkg_data_columns is included in geopackage, use it's description field to supplement the schema.
+	schemaDataColumns, err := hasSchemaDataColumnsTable(db)
+	if err != nil {
+		return nil, err
+	}
+
 	rows, err := db.Queryx(fmt.Sprintf("select name, type, \"notnull\" from pragma_table_info('%s')", table.TableName))
```

Comment:
ipv aparte db.Get call per row. Zou ik ervoor kiezen om het in 1 query op te lossen via een join. Dus als `hasSchemaDataColumnsTable==false` dan blijft de query zoals 'ie nu is. Is er wel sprake van een schema extension dan de query met een join op `gpkg_data_columns` uitvoeren.

Dit is niet enorm performance sensitive code aangezien het 1x tijdens startup wordt uitgevoerd. Maar als dit wel vaker zou worden uitgevoerd dan zou dit een flinke performance hit zijn. Dit specifieke patroon (query uitvoeren en daarop weer X andere queries) staat bekend als het "N+1 select" probleem.